code/modules/jobs/job_types/job.dm

```diff
@@ -188,3 +189,8 @@
 		for(var/implant_type in implants)
 			var/obj/item/weapon/implant/I = new implant_type(H)
 			I.implant(H, null, silent=TRUE)
+
+	if(organs)
+		for(var/organ_type in organs)
+			var/obj/item/organ/O = new organ_type(H)

```

Comment:
still no reason to spawn it in the person, really.
that actually has the implication that they ARE in the mob (when they aren't supposed to be), if someone read this code they'd get confused.

just ```var/obj/item/organ/O = new organ_type``` will do.